packages/core/src/client/api.ts

```diff
@@ -69,9 +69,9 @@ export interface RelationPartInclusion {
 /** A claim that the same data is referred to by another CID and/or multihash */
 export interface EqualsClaim extends ContentClaim<typeof Assert.equals.can> {
   /** Any CID e.g a CAR CID */
-  readonly content: Link
+  readonly content: UnknownLink
```

Comment:
type fix... a v0 CID can be equivalent to another CID, so the types should allow UnknownLink here.